GameServerLibTests/Tests/Items/TestInventoryManager.cs

```diff
@@ -71,7 +75,7 @@ public void TestItemStacking()
             Assert.AreEqual(1, item2.StackCount);
 
             // Stack the second item, and make sure the second gets stacked
-            for(var i = 0; i < itemType2.MaxStack - 1; i++)
+            for (var i = 0; i < itemType2.MaxStack - 1; i++)
```

Comment:
Why did you add space here?

Reply:
Above it was done the same, so I changed it for consistency.